docs/includes/framework-compatibility-laravel.rst

```diff
@@ -11,6 +11,7 @@
      -
 
    * - 4.0
-     - ✓
+     - ✓ [#min-version-note]_
      -
 
+.. [#min-version-note] {+odm-short+} v4.0 is compatible with Laravel v9.3.9 and later.
```

Comment:
This is wrong. The package `mongodb/laravel-mongodb` version 4.0 is compatible with Laravel 10 only. This is described [in composer.json](https://github.com/mongodb/laravel-mongodb/blob/4.0/composer.json#L27-L30).